packages/common-common/src/api/extApiTypes.ts

```diff
@@ -139,6 +139,7 @@ export type GetRolesReq = {
   community_id: string;
   addresses?: string[];
   count_only?: boolean;
+  permissions?: string[];
```

Comment:
Reason for this change?

Reply:
It the `permissions` query param allows filtering roles by the permissions level, e.g. `admin` or `moderator`. Otherwise it returns roles for all users of the chain.